CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - [#4375](https://github.com/influxdb/influxdb/pull/4375): Add Subscriptions so data can be 'forked' out of InfluxDB to another third party.
 - [#4506](https://github.com/influxdb/influxdb/pull/4506): Register with Enterprise service and upload stats, if token is available.
 - [#4501](https://github.com/influxdb/influxdb/pull/4501): Allow filtering SHOW MEASUREMENTS by regex.
+- [#4547](https://github.com/influxdb/influxdb/pull/4547): Drop raft node.
```

Comment:
Just so the guys reading the CHANGELOG know what this about is about can you rephrase it as something like:

`DROP SERVER` for all node types (or something).